gtfs/spec/en/reference.md

```diff
@@ -380,6 +382,30 @@ Describes levels in a station. Useful in conjunction with `pathways.txt`, and is
 |  `level_index` | Float | **Required** | Numeric index of the level that indicates its relative position. <br><br>Ground level should have index `0`, with levels above ground indicated by positive indices and levels below ground by negative indices.|
 |  `level_name` | Text | Optional | Name of the level as seen by the rider inside the building or station.<hr>_Example: Take the elevator to "Mezzanine" or "Platform" or "-1"._|
 
+### occupancies.txt
+
+File: **Optional**
+
+Usual or expected in-vehicle occupancy levels.
+
+For describing historical average occupancies at least 4 weeks of occupancy data should be averaged across each unique day of the week, valid for up to 2 weeks thereafter in `occupancies.txt`.
+
+Methods for occupancy forecasting used to populated `occupancies.txt` are not strictly specified. Beyond the guidelines given above, it is the responsibility of data producers to ensure that the data provided are meaningful. 
+
+| Field Name | Type | Required | Description |
+| ----- | ----- | ----- | ----- |
+| `trip_id` | ID referencing `stop_times.trip_id` | **Required** |  Identifies a `trip_id` for which an occupancy level is described. |
```

Comment:
Should this be referencing `trips.trip_id`? How does this interact with `occupancies.stop_sequence` if both are defined?